README.md

```diff
@@ -6,17 +6,25 @@ This project started as a python port of Camat (cardiac mapping analysis tool, P
 In order to get up and running with Kairosight 2.0 you will need to complete the following set up steps:
 1. First you will need to install Anaconda, which can be found [here](https://docs.anaconda.com/anaconda/install/windows/).
 2. Clone or download the repository
-3. On a Windows PC follow steps 4-5, for Mac skip to steps 6-9.
+3. On a Windows PC follow steps 4-6, for Mac skip to steps 7-10.
 4. Open the Anaconda Prompt and navigate to the directory where you cloned/downloaded the repository (e.g., "cd OneDrive\Documents\GitHub\kairosight-2.0")
 5. Enter the following command to setup the Anaconda environment: `conda env create -f kairosight_env.yml`
-6. Open terminal
-7. Enter the following command: 'conda create -n kairosight_env.yml'
-8. Proceed with 'y'
-9. Enter the following command: 'conda activate kairosight_env.yml'
-10. Close the Anaconda Prompt
-11. Launch Anaconda Navigator and switch to the newly created environment
-12. Launch Spyder
-13. In the top menu select: Tools -> Preferences
-14. Select IPython console on the left hand menu
-15. Select the Graphics tab and make sure the Graphics backend is set to Qt5
-16. Open kairosight_retro.py and hit the play button 
+6. Close the Anaconda Prompt
+7. Open terminal
+8. Enter the following command: 'conda create -n kairosight_env.yml'
```

Comment:
When you create a conda environment like this, the name of the environment is defined by the first line of the yaml file.

Reply:
I looked into it more and you're correct. I need to check the set up again, this could be why I needed to install so many packages when I ran spyder. Thank you for checking this!

Reply:
The other potential issue is that the anaconda prefix path is hardcoded at the end of the yaml file. (not part of this PR or your fault, it was like that before). You should probably update that when you get the chance.